### DISCUSSION TITLE
VCRUNTIME140_1.dll was not found

### DISCUSSION THREAD

**Discussion by ea5wa** [foldynl/QLog#287] [Q&A]
Trying to install QLog in a new windows 11 installation (Tiny version) I have an error after first run of QLog.

![image](https://github.com/foldynl/QLog/assets/54686689/1a58cd20-87cd-467c-82fb-6c8d846a90eb)

I have repaired visual c++ 2015 redistributable, but still same message.
Any help is appreciated

**Comment by foldynl** [comment]
Did you try to install [this](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022)? Did you restart Windows after the installation?

**Comment by ea5wa** [comment]
After installing SDRUno and SDR Console V3, several visual c++ redistributable has been installed and now QLog is running

**Comment by Suitbert** [comment]
Hello, I got the [MSVCP140_2.dll was not found] error while installing QLog on a new Windows 11 PC today. Visual c++ redistributable has been installed allready. What can I do now? Please help, Thanks.
